src/components/remote_control/src/remote_control_plugin.cc

```diff
@@ -243,6 +243,35 @@ void RemoteControlPlugin::NotifyMobiles(
   }
 }
 
+void RemoteControlPlugin::ProcessApplicationPolicyUpdate() {
+  typedef std::vector<application_manager::ApplicationSharedPtr> Apps;
+  Apps app_list = service()->GetApplications(GetModuleID());
+  Apps::const_iterator app = app_list.begin();
+  for (; app_list.end() != app; ++app) {
+    const uint32_t application_id = (*app)->app_id();
```

Comment:
@LuxoftAKutsan 
Is it matter of personal preferences or there is some convention?